bluebrain/repo-patches/packages/neuron/package.py

```diff
@@ -21,434 +20,288 @@ class Neuron(CMakePackage):
-    url = "http://www.neuron.yale.edu/ftp/neuron/versions/v7.5/nrn-7.5.tar.gz"
+    url = "https://github.com/neuronsimulator/nrn/releases/download/8.2.3/full-src-package-8.2.3.tar.gz"
     git = "https://github.com/neuronsimulator/nrn"
-
-    # Patch which reverts 81a7a39 for numerical compat
-    patch("revert_Import3d_numerical_format.master.patch", when="@7.8.1:9.0.a6")
-
-    # Patch which reverts d9605cb for not hanging on ExperimentalMechComplex
-    # Patch for recent CMake versions that don't identify NVHPC as PGI
-    patch("patch-v800-cmake-nvhpc.patch", when="@8.0.0%nvhpc^cmake@3.20:")
+    maintainers("pramodk", "nrnhines", "iomaganaris", "ohm314", "matz-e")
 
     version("develop", branch="master")
-    version("9.0.a11", commit="b38dd11b")
-    version("9.0.a10", commit="af51e51e")
-    version("9.0.a9", commit="baf3b14")
-    version("9.0.a8", commit="67a672a")
-    version("9.0.a5", commit="522c866")
-    version("9.0.a4", commit="de2c927")
-    version("9.0.a3", commit="afce1ef")
-    version("9.0.a2", commit="89f7dab")
-    version("9.0.a1", commit="b3c4b4f")
-    version("8.2.2", tag="8.2.2")
-    version("8.2.1", tag="8.2.1")
+    # TODO: 9.0 for testing purposes, remove it
+    version("9.0", commit="53c9392d7")
+    version("8.2.3", tag="8.2.3")
     version("8.2.0", tag="8.2.0")
     version("8.1.0", tag="8.1.0")
-    version("8.0.2", tag="8.0.2")
-    version("8.0.1", tag="8.0.1")
     version("8.0.0", tag="8.0.0")
-    version("7.8.1", tag="7.8.1")
+    version("7.8.2", tag="7.8.2")
 
-    variant(
-        "binary",
-        default=True,
-        description="Create special as a binary instead of shell script (8.0.x and earlier)",
-        when="@:8.0.999",
-    )
+    # neuron variants for basic installation
+    variant("interviews", default=False, description="Enable GUI with INTERVIEWS")
+    variant("legacy-unit", default=False, description="Enable legacy units")
+    variant("mpi", default=True, description="Enable MPI parallelism")
+    variant("python", default=True, description="Enable python")
+    variant("tests", default=False, description="Enable building tests")
+    variant("rx3d", default=False, description="Enable cython translated 3-d rxd.", when="+python")
+
+    # variants from coreneuron support
+    variant("coreneuron", default=True, description="Enable CoreNEURON support")
+    variant("gpu", default=False, description="Enable GPU build", when="@9:+coreneuron")
+    variant("openmp", default=False, description="Enable CoreNEURON OpenMP support", when="+coreneuron")
+ 
+    # instrumentation for performance measurement
+    variant("caliper", default=False, description="Add Caliper support")
+
+
+    #################### ALL OF THE BELLOW TO BE DISCUSSED & DELETED #######################
+
+    # add build type 'FastDebug' for result validations
+    # TODO: FastDebug should be removed? Maybe useful for internal tests
+    #       and not BBP CI setup. So should be OK to keep?
```

Comment:
Since the default is reasonable this won't hurt to just stay as an option.

Reply:
I think that this could be helpful to keep in the BBP package only for the ease of use when we have to build neurodamus-packages for reproducing issues, CI, etc